docs/api.md

```diff
@@ -11,7 +11,7 @@ hide_title: true
 Functions marked with {🚀} are considered advanced, and should typically not be needed.
 Consider downloading our handy cheat sheet that explains all important APIs on a single page:
 
-<div class="cheat"><a href="https://gum.co/fSocU"><button title="Download the MobX 6 cheat sheet and sponsor the project">Download the MobX 6 cheat sheet</button></a></div>
+<div class="cheat"><a href="https://gum.co/fSocU"><button title="Download the MobX 6 cheat sheet and sponsor the project">Get the MobX 6 cheat sheet (£5)</button></a></div>
```

Comment:
This was something that I had found to be unexpected when I was first reading the documentation, since there weren't any indications of it requiring payment. Not a huge deal, but I believe that knowing beforehand which links require payment is more in line with reader's expectations.